src/BusinessChat/Enums.cs

```diff
@@ -0,0 +1,27 @@
+using XamCore.ObjCRuntime;
+using XamCore.Foundation;
+
+namespace XamCore.BusinessChat {
+
+	[Mac (10,13, onlyOn64: true), iOS (11,3)]
```

Comment:
pretty sure it did not exists in 10,13
let's put 10,13,4 which is the SDK where it became public